src/alphabetical.py

```
@@ -0,0 +1,553 @@
+def main():
+    # Open with explicit utf-8
+    with open("README.md", "r", encoding="utf-8") as file:
+        orig_lines = file.readlines()
+
+    # make a small backup so changes are reversible
+    try:
+        with open("README.md.bak", "w", encoding="utf-8") as bfile:
+            bfile.writelines(orig_lines)
+    except Exception:
+        # non-fatal if backup can't be written
+        pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)